clients/admin-ui/src/features/user-management/PasswordManagement.tsx

```diff
@@ -39,14 +34,12 @@ const PasswordManagement = ({ profileId }: Props) => {
     return null;
   }
 
-  if (!isOwnProfile && !canForceResetPassword) {
-    return null;
-  }
-
   return (
     <HStack>
       {isOwnProfile ? <UpdatePasswordModal id={profileId} /> : null}
-      {canForceResetPassword ? <NewPasswordModal id={profileId} /> : null}
+      <Restrict scopes={[ScopeRegistry.USER_PASSWORD_RESET]}>
```

Comment:
example of `<Restrict />` in action! cleans up a good amount of boilerplate code